### DISCUSSION TITLE
July 25 2023 Stand Up

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#419] [Q&A] [answered]
As titled. =)

**Comment by hollowaykeanho** [comment]
Wait what? I thought you are onto job search or something?

**Comment by corygalyna** [comment]
habits die hard. Besides, I still miss our collaborations.

**Comment by hollowaykeanho** [comment, marked as answer]
CURRENTLY: deploying Hestia as a PWA app on the official site. 

NEXT: publish all the Hestia papers and backlink them to the official sites.

**Comment by corygalyna** [comment]
Help?

**Comment by hollowaykeanho** [comment]
Not that much. =)

**Comment by corygalyna** [comment]
Ok got it.
